packages/neos-ui-contentrepository/src/registry/NodeTypesRegistry.ts

```diff
@@ -215,6 +218,20 @@ export default class NodeTypesRegistry extends SynchronousRegistry<NodeType> {
                                     helpThumbnail: property.ui?.help?.thumbnail
                                 })
                             ),
+                        ...references.filter(p => p.ui?.inspector?.group === group.id)
+                                .map(property => ({
+                                    type: 'editor',
+                                    id: property.id,
+                                    label: property.ui?.label,
+                                    editor: property.ui?.inspector?.editor,
+                                    editorOptions: property.ui?.inspector?.editorOptions,
+                                    position: property.ui?.inspector?.position,
+                                    hidden: property.ui?.inspector?.hidden,
+                                    helpMessage: property.ui?.help?.message,
+                                    helpThumbnail: property.ui?.help?.thumbnail,
+                                    properties: property?.properties
```

Comment:
ah we just expose it here but its not interpreted anywhere? I think we should use a better term here ...

Reply:
Yes, would be a follow-up to use it in the editor, for now it can be used anywhere. 

Would perhaps edgeProperties be better?

Reply:
or `referenceProperties`? Maybe introduce this thing first with the mentioned followup and not now? ... on the other hand its just the yaml mapped 1to1 so we should probably not mess with it too much? ... but its not 1to1 really so maybe `referenceProperties`? :D

Reply:
`referenceProperties` works for me, it is also used fairly often in the documentation: https://docs.neos.io/guide/content-repository/nodetype/references-vs-properties 
I can also remove it for now, it does not really make a difference if we add it now or later.

Reply:
> it does not really make a difference if we add it now or later.

true in this case its simple ... but id do it from time to time to delay the discussion and let the reviewers focus more on the core parts :D ... Id vote for removing it now as without using this thing we dont know what the editor actually needs.
For example this pr would just pass the properties 1to1 but maybe they should also be mapped and get some view configuration? For people already wanting to experiment, there are alternative ways to fetch the reference properties by getting the js nodetype so i think that would be fine :D

Reply:
I removed it for now 👍